Cargo.toml

```diff
@@ -77,15 +77,16 @@ tracing = { version = "0.1.40", features = ["log"] }
 yaxpeax-arm = "0.3.0"
 chrono = "0.4.38"
 vergilius = "0.1.2"
-windows = { version = "0.57.0", features = [
-    "Win32_Foundation",
+windows-sys = { features = [
+    "Win32",
     "Win32_System",
     "Win32_System_SystemServices",
     "Win32_System_Diagnostics_Debug",
     "Win32_System_Diagnostics",
     "Win32_System_SystemInformation",
     "Win32_System_Kernel",
-] }
+    "Win32_Foundation",
+], version = "0.59.0" }
```

Comment:
Can reduce some of the duplicity here, thanks to the crate's feature hierarchy. For example, `Win32_System_Diagnostics_Debug` brings in `Win32_System_Diagnostics` which brings in `Win32_System`..., etc. 

```toml
[dependencies.windows-sys]
version = "0.59.0"
features = [
  "Win32_Foundation",
  "Win32_System_Diagnostics_Debug",
  "Win32_System_Kernel",
  "Win32_System_SystemInformation",
  "Win32_System_SystemServices",
]

```

Reply:
Thanks for the tip Rafael!